conf/routes

```diff
@@ -80,7 +80,8 @@ POST   /researcher/v1/study/participants @controllers.StudyController.sendStudyP
 POST   /researcher/v1/study              @controllers.StudyController.updateStudyForResearcher
 
 # Researchers - Upload Schemas
-GET    /researcher/v1/uploadSchema/:schemaId  @controllers.UploadSchemaController.getUploadSchema(schemaId: String)
+GET    /researcher/v1/uploadSchema/byId/:schemaId  @controllers.UploadSchemaController.getUploadSchema(schemaId: String)
+GET    /researcher/v1/uploadSchema/forStudy   @controllers.UploadSchemaController.getUploadSchemasForStudy
```

Comment:
I guess thinking about the resources I would have:
/researcher/v1/uploadSchemas/:schemaId
/researcher/v1/study/uploadSchema
or possibly
/researcher/v1/uploadSchemas/study
I would also not upper-case the URL since URLs are case insensitive.